wiki/Tournaments/MCNC/7K2023/en.md

```diff
@@ -0,0 +1,470 @@
+  8. [Wang Qiang - Qiu Tian Bu Hui Lai \[Sorrowful Love\]](https://osu.ppy.sh/beatmapsets/1516775#mania/3105133)
+  9. [Sengoku Nadeko (CV: Hanazawa Kana) - Mousou Express (Cut Ver.) \[Imaginary Nerf\]](https://osu.ppy.sh/beatmapsets/1922894#mania/3969432)
+  10. [Schubert - Introduction And Variations 'Trockne Blumen' Variation 5 Remix&Arrange \[Blocko's 7K Irrsinnig\]](https://osu.ppy.sh/beatmapsets/557139#mania/1507217)
+  11. [Yunomi & nicamoq - Indoor Kei Nara Trackmaker \[= w=)zzz\]](https://osu.ppy.sh/beatmapsets/833226#mania/1745510)
+- TieBreaker
+  1. [Teminite & MDK - Space Invaders \[8bit Revenger // co. _Stan\]](https://osu.ppy.sh/beatmapsets/1922871#mania/3969384)
+
+### Round of 32
+
+**[Download the mappack here! (77 MB)](https://drive.google.com/file/d/1olG5EJrn0Dfuy1A1IDElMw1nbSDVWff6/view?usp=sharing)**
+
+- FreeMod
+  1. [you - Hold Angel \[Insane\]](https://osu.ppy.sh/beatmapsets/636543#mania/1351134)
+  2. [qfeileadh feat. Lycoris Radiata - Soumai Musou Lovers \[Insane\]](https://osu.ppy.sh/beatmapsets/1758665#mania/3599277)
+  3. [paraoka - L9 \[7K Another\]](https://osu.ppy.sh/beatmapsets/1565681#mania/3256342)
+  4. [guna with team kyukkyu - subconsciousness \[nyarather\]](https://osu.ppy.sh/beatmapsets/1918757#mania/3959554)
+  5. [Freeleaves - Wisteria \[longevity\]](https://osu.ppy.sh/beatmapsets/1536959#mania/3142518)
+  6. [Xacla - oceanbird \[less ln lock\]](https://osu.ppy.sh/beatmapsets/1918754#mania/3959550)
+  7. [Shimotsuki Haruka - Re:Call \[KaffuChino's Re:Call (edit)\]](https://osu.ppy.sh/beatmapsets/1918773#mania/3959583)
+  8. [Various - Kim_GodSSI's Guest Pack 2 \[M2U & Nicode - Stellar\]](https://osu.ppy.sh/beatmapsets/868254#mania/1814871)
+  9. [Linear ring - Enchanted love \['o'\]](https://osu.ppy.sh/beatmapsets/1349017#mania/2793207)
+- TieBreaker
+  1. [t+pazolite - Ultrasonic Anthem \[Expert // co. tyrcs\]](https://osu.ppy.sh/beatmapsets/1918758#mania/3959555)
+
+### Qualifiers
+
+**[Download the mappack here! (34 MB)](https://drive.google.com/file/d/1rd_tYLdZSxli9xp1fPIq8IOAKl-jsaya/view?usp=sharing)**
+
+- FreeMod
+  1. [Kirara Magic - Zombie Step \[Stage 1: Halowin\]](https://osu.ppy.sh/beatmapsets/1914412#mania/3949437) |
+  2. [MOB CHOIR - 99.9 (Schwank bootleg) \[Stage 2: 100%\]](https://osu.ppy.sh/beatmapsets/1914499#mania/3949603) |
+  3. [FROL - BEAKER (roop remix) \[Stage 3: Backtrack\]](https://osu.ppy.sh/beatmapsets/1914413#mania/3949438) |
+  4. [awfuless meets Zekk - MATOI -phase L- \[Stage 4: Showdown\]](https://osu.ppy.sh/beatmapsets/1914417#mania/3949442) |
+  5. [Asaka - Open your eyes  (Sped Up Ver.) \[Stage 5: Psychic\]](https://osu.ppy.sh/beatmapsets/1914415#mania/3949440) |
+  6. [EZFG - Cyber Thunder Cider \[Stage 6: PPPP\]](https://osu.ppy.sh/beatmapsets/1914416#mania/3949441) |
+  7. [technoplanet - Awakening \[Long Version\] [Stage 7: Ascension]](https://osu.ppy.sh/beatmapsets/1914418#mania/3949443) |
+
+## Match results
```

Comment:
dates are missing from all stages. please see the template: https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md

example: https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/OWC/2022/en.md?plain=1#L322

Reply:
fixed in https://github.com/ppy/osu-wiki/pull/10380/commits/795eba5f427760615b0ea48f52235942e02e1e32